src/ecalc/libraries/libecalc/common/libecalc/common/string_utils.py

```diff
@@ -21,3 +21,17 @@ def generate_id(*args: str) -> str:
     """
     full_string = "-".join(args)
     return hashlib.md5(full_string.encode()).hexdigest()  # noqa: S324 - insecure hash for ids
+
+
+def to_camel_case(string: str) -> str:
+    """Convert string from snake_case to camelCase
+
+    Args:
+        string: String in snake_case format
+
+    Returns:
+        String in camelCase format
+
+    """
+    string_split = string.replace("__", "_").split("_")
```

Comment:
den ser ut til å gjøre jobben sin..for det meste..men muligens ikke den best to_camel_case funksjonen som eksisterer...

Reply:
Bedre enn den som var i fde :)